runtime/major_gc.c

```diff
@@ -405,7 +502,17 @@ Caml_inline void mark_slice_darken(struct mark_stack* stk, value v, mlsize_t i,
   child = Field (v, i);
 
 #ifdef NO_NAKED_POINTERS
-  if (Is_block (child) && !Is_young (child)) {
+  if (Is_block (child)
+      && ! Is_young (child)) {
```

Comment:
Similarly - it'll help easing into the review upstream if the green bits are clearly `#ifdef NAKED_POINTERS_CHECKER`!